pages/delegates/index.tsx

```diff
@@ -375,6 +395,24 @@ const Delegates = ({
                   ))}
                 </Stack>
               )}
+
+              {delegates.length && (
```

Comment:
haven't tested it, but I assume if delegates.length === 0 this will return false? Also do we need to check if delegates is undefined?

Reply:
My bad, I was going to change that line as that syntax is ambiguous. But yeah, as 0 is a falsy value that evaluates to false and that JSX is skipped.
Theoretically, I think delegates can't be undefined as the state variable is always set to an array, either empty or with values. What I'm seeing is that the page may break if the API call fails (delegates returned from the API would be undefined and when using the spread operator on undefined it throws an error).

I need to re-check if the API endpoint throws an error on failure or if it returns empty keys